src/main/java/io/stargate/sgv2/jsonapi/config/feature/ApiFeatures.java

```diff
@@ -51,4 +51,18 @@ public boolean isFeatureEnabled(ApiFeature flag) {
     }
     return flag.enabledByDefault();
   }
+
+  private Boolean booleanFromString(String str) {
+    if (str == null || str.isBlank()) {
+      return null; // no value, so not enabled
+    }
+    if ("true".equals(str)) {
```

Comment:
should this be `equalsIgnoreCase()`

Reply:
Matter of taste: I think by default only "true" and "false" accepted for boolean/Boolean in Java.